docs/QSG/docker.md

```diff
@@ -1,172 +1,87 @@
+
+@@@ Is this a warning about rocky and huge pages or daos with hugepages on rocky or??? 
 !!! warning
-    Virtual Docker network such as [bridge](https://docs.docker.com/network/bridge/) are not yet
-    well supported by DAOS.  Thus, one physical network interface (i.e. loopback interface is also
-    not well supported) of the host should be chosen for being used by the containers through the
-    Docker [host](https://docs.docker.com/network/host/) network.
-
-
-## Prerequisites
-
-To build and deploy the Docker images, `docker` and optionally `docker-compose` shall be available.
-The docker host should have access to the [Docker Hub](https://hub.docker.com/) and
-[Rocky Linux](https://rockylinux.org/) official repositories.  Finally,
-[hugepages](https://www.kernel.org/doc/Documentation/vm/hugetlbpage.txt) linux kernel feature shall
-be enabled on the docker host.  At least, 4096 pages of 2048kB should be available.  The number of
-huge pages allocated could be checked with the following command:
-
-```bash
-$ sysctl vm.nr_hugepages
-```
-
-The default size of a huge page, the number of available huge pages, etc. could be found with the
-following command:
-
-```bash
-$ cat /proc/meminfo | grep -e "^Huge"
-```
-
-The platform was tested and validated with the
-[rockylinux/rockylinux:8.4](https://hub.docker.com/r/rockylinux/rockylinux) and
-[centos:centos8](https://hub.docker.com/_/centos) official docker images.  However other RHEL-like
-distributions should be supported.
-
-!!! warning
-    Some distributions are not yet well supported such as
-    [rockylinux/rockylinux:8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the
-    management of hugepages with the [spdk](https://spdk.io/) library.
-
-
+    Some distributions are not yet well supported such as [rockylinux/rockylinux:8.5](https://hub.docker.com/r/rockylinux/rockylinux): issue with the management of hugepages with the [spdk](https://spdk.io/) library.
 
 ### Configuring HugePages
-
-First the Linux kernel needs to be built with the `CONFIG_HUGETLBFS` (present under "File systems")
-and `CONFIG_HUGETLB_PAGE` (selected automatically when `CONFIG_HUGETLBFS` is selected) configuration
-options.
-
-To avoid memory fragmentation, huge pages could be allocated on the kernel boot command line by
-specifying the "hugepages=N" parameter, where 'N' = the number of huge pages requested.  It is also
-possible to allocate them at run time, thanks to the `sysctl` command:
-
-```bash
-$ sysctl vm.nr_hugepages=8192
-```
-
-It is also possible to use the `sysctl` command to allocate huge pages at boot time with the
-following command:
-
-```bash
-$ cat <<< "vm.nr_hugepages = 8192" > /etc/sysctl.d/50-hugepages.conf
-$ sysctl -p
-```
+If hugepages have not been enabled, we will need to do so now to avoid memory fragmentation. To configure 2Mb hugepages use the following command:
```

Comment:
Running this command alone will not be enough. The host machine should also be rebooted.

Reply:
commented